src/components/UsersStack/UsersStack.tsx

```diff
@@ -129,26 +132,31 @@ export const UsersStack = ({
           const direction =
             i === 0 && !canShowOthers ? "circle" : directionClip;
 
+          const id = `UsersStack__defs--${cmpId}-${i}`;
+          const hrefID = `#${id}`;
+          const maskID = `UsersStack__mask--${cmpId}-${i}`;
```

Comment:
```suggestion
          const id = `UsersStackDefs${cmpId}${i}`;
          const hrefID = `#${id}`;
          const maskID = `UsersStackMask${cmpId}${i}`;
```

Мб так, чтобы не сэкономить байтов? По идее можно ещё жосче сократить. Тут ведь семантика не нужна, тема для внутреннего использования